src/libs/actions/IOU.js

```diff
@@ -1068,6 +1070,8 @@ function requestMoney(
     receipt = undefined,
     category = undefined,
     tag = undefined,
+    taxCode = '',
+    taxAmount = 0,
```

Comment:
Should this be undefined like other values?

Reply:
Uhm, yea right - but when we don't have tax policies defined, we want to send in those values by default. it seems I'm missing the point [here](https://docs.google.com/document/d/1p2RP7xqMWVK1RMOiSLdGl9L8N1qyjFOStwFOY9sfqIQ/edit#heading=h.wdi70gzmr3q)

Reply:
All good 👍